### ISSUE TITLE
contact info

### ISSUE THREAD

**Issue by 0neBeerLeft** [UnrealNoodle/AntiVM#1]
do you have telegram?

**Comment by UnrealNoodle** [comment]
Yes but I don't have it now, I am out for vacations. I'll be back after 2 months but I will try using telegram in the coming week.
